mappings/net/minecraft/client/render/model/BakedQuadFactory.mapping

```diff
@@ -1,5 +1,59 @@
 CLASS net/minecraft/class_796 net/minecraft/client/render/model/BakedQuadFactory
+	FIELD field_4259 MAX_SCALE F
+	FIELD field_4260 MIN_SCALE F
 	METHOD method_3454 uvLock (Lnet/minecraft/class_787;Lnet/minecraft/class_2350;Lnet/minecraft/class_4590;Lnet/minecraft/class_2960;)Lnet/minecraft/class_787;
+		ARG 0 texture
+		ARG 1 orientation
+		ARG 2 rotation
+		ARG 3 modelId
+	METHOD method_3455 transformVertex (Lnet/minecraft/class_1160;Lnet/minecraft/class_4590;)V
+		ARG 1 vertex
+		ARG 2 transformation
+	METHOD method_3456 getLightBrightness (Lnet/minecraft/class_2350;)F
+		ARG 1 direction
+	METHOD method_3457 getBrightnessInteger (Lnet/minecraft/class_2350;)I
```

Comment:
Is the brightness integer the light map coordinate (block and sky light combination)?

Reply:
No, I don't think it's the same as world light values. These are the 3D lighting effects applied to models, which vary per face.

`getLightBrightness` is the relative brightness for a given face by direction.
`getBrightnessInteger` is the result of `getLightBrightness` packed into an integer value.

```
    private int getBrightnessInteger(Direction direction) {
        int brightness = MathHelper.clamp((int)(this.getLightBrightness(direction) * 255), 0, 255);
        return 0xFF000000 | brightness << 16 | brightness << 8 | brightness;
    }
    
    private float getLightBrightness(Direction direction) {
        switch (direction) {
            case DOWN: return 0.5F;
            case UP: return 1F;
            case NORTH:
            case SOUTH: return 0.8F;
            case WEST:
            case EAST: return 0.6F;
            default: return 1F;
        }
    }
```

It's calculated and passed down as one of the components that are packed into the vertex buffer thing.

Here:
```
int brightness = shaded ? this.getBrightnessInteger(direction11) : -1;
```

And here:
```
        vertices[integer8] = Float.floatToRawIntBits(position.getX());
        vertices[integer8 + 1] = Float.floatToRawIntBits(position.getY());
        vertices[integer8 + 2] = Float.floatToRawIntBits(position.getZ());
        vertices[integer8 + 3] = brightness;
        vertices[integer8 + 4] = Float.floatToRawIntBits(sprite.getU(texture.getU(cornerIndex)));
        vertices[integer8 + 4 + 1] = Float.floatToRawIntBits(sprite.getV(texture.getV(cornerIndex)));
```

Reply:
This is called `light` in other places. It's the uv coordinates on the lightmap.